service/models/messages.js

```diff
@@ -1,12 +1,17 @@
 const mongoose = require("mongoose");
 
 const messageSchema = new mongoose.Schema({
-    to: {type: String, required: true},
-    from: {type: String, required: true},
-    subject: {type: String, required: true},
-    message: {type: String, required: true},
+    from: { type: String, required: true },
+    subject: { type: String, required: true },
+    message: { type: String, required: true },
+    timestamp: { type: Date, default: Date.now },
```

Comment:
Remember that you'll be responsible for passing and parsing dates between client and server, which is a challenge in JavaScript.